internal/framework/service/rulesets/resource.go

```diff
@@ -73,6 +75,13 @@ func (r *RulesetResource) Create(ctx context.Context, req resource.CreateRequest
 	zoneID := data.ZoneID
 	rulesetPhase := data.Phase.ValueString()
 
+	if rulesetPhase == sbfmPhaseName {
```

Comment:
instead of adding a new constant, we can also use `cloudflre.RulesetPhaseSuperBotFightMode` here.